var/spack/repos/builtin/packages/scine-qcmaquis/package.py

```diff
@@ -15,42 +17,63 @@ class ScineQcmaquis(CMakePackage):
     maintainers("adam-grofe")
 
     version("master", branch="master")
+    version("3.1.4", branch="release-3.1.4")
     version("3.1.3", branch="release-3.1.3")
-    version("3.1.2", branch="release-3.1.2", preferred=True)
-    variant(
```

Comment:
Is the removal of this variant intentional?  Does it not still apply to at least the older versions?

Reply:
The `blas` variant was used to select the BLAS/LAPACK library to use. However, it does not define any input for Cmake, which detects this library automatically. QCMaquis does not specify a default/preferred one.
The user can decide whether to use OpenBlas or MKL by specifying `^openblas` or `^intel-oneapi-mkl` in the spec, or they can leave the choice to spack's concretizer. So I do not see the need for a `blas` variant.
Let me know what is Spack's suggested approach in this case.